src/color-modes/native.scss

```diff
@@ -9,3 +9,11 @@
   color: var(--color-fg-default);
   background-color: var(--color-canvas-default);
 }
+
```

Comment:
Thanks for upstreaming. 🙇  Mind adding [these comments](https://github.com/github/github/blob/b280d5bc529b03cc1721434a4cd74af3fde8a566/app/assets/stylesheets/hacks/hx_color-modes.scss#L152-L154) here too?

Just so it's easier to remember what this does. 😃